src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +15,10 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        User user = userService.findByEmail(email);
+        if (user != null) {
+            return user.getPassword().equals(password);
+        }
         return false;
```

Comment:
```suggestion
        user != null && user.getPassword().equals(password);
```
Let's simplify this construction :)